dev/MRTCore/mrt/NuGet/Microsoft.ProjectReunion.nuspec

```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
this file should not be under MRTCore folder

Reply:
I think this is more or less true I wonder when we will have a plan for aggregating multiple nuspecs for the sub modules inside of the Project Reunion repo into one we can feed to the nuget pack command.  I think we don't have this plan yet so it's probably OK that it is here temporarily.

Reply:
You two can work out what name this should have.

Reply:
I also asked this question, and the answer is that MRTCore doesn't have an identity outside of Project Reunion, so it shouldn't get it's own nuget package.  The assertion is that there won't be anyone using MRTCore without using all of ProjectReunion (aside from WinUI who will do something special).

Reply:
when Reunion grows, say it contains 100 subprojects. It's very unlikely someone will need them all.

Reply:
@EHO-makai would you like to take this?

Reply:
I handled this in email, there is a doc (which will be live on github so everyone can see) that explains the packaging plan.  Basically there will be "thin" nuget packages from the various repos that aggregate into Project Reunion and Project Reunion's "metapackage" will reference all of these.  So this repo outputs one "thin" nuget package with metadata and VS targets files, etc. and one metapackage that references everything in Project Reunion.  The actual binaries are aggregated into one framework package that gets used at runtime.

Reply:
"thin" == metadata plus binaries (dll's), "fat" == contains dll, Project Reunion contributors will publish thin nuget packages to public nuget.org.  The binaries live in the Project Reunion framework package.

Reply:
Awesome, thanks Eho.